src/main/java/com/linkmoa/domain/member/entity/Role.java

```diff
@@ -0,0 +1,5 @@
+package com.linkmoa.domain.member.entity;
+
+public enum Role {
+	ROLE_USER,ROLE_ADMIN
```

Comment:
이미 열거형 이름에 `Role`이 있어서 `Role.ROLE_ADMIN`은 좀 이상해 보이는 것 같습니다.

```suggestion
    USER, ADMIN;

    public String getRoleName() {
        return "ROLE_" + name();
    }
```